docs/building-blocks/extending-templates/conditional-deploy.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Conditionally deploy a resource in an Azure Resource Manager template
+description: Describes how to extend the functionality of Azure Resource Manager templates to conditionally deploy a resource dependending on the value of a parameter
+author: petertay
+ms.date: 06/09/2017
+
+---
+
+# Conditionally deploy a resource in an Azure Resource Manager template
+
+There are some scenarios in which you need to design your template to deploy a resource based on a condition, such as whether or not a parameter value is present. For example, your template may deploy a virtual network and include parameters to specify other virtual networks for peering. If you've not specified any parameter values for peering, you don't want Resource Manager to deploy the peering resource.
+
+To accomplish this, use the [`condition` element][azure-resource-manager-condition] in the resource to test the length of your parameter array. If the length is zero, return `false` to prevent deployment, but for all values greater than zero return `true` to allow deployment.
```

Comment:
Did you really mean to have [`condition` element]? It renders weird. Should you remove the `s